src/main/resources/data/minecolonies/tags/blocks/pathblocks.json

```diff
@@ -6,6 +6,17 @@
     "minecraft:stone_brick_stairs",
     "minecraft:stone_brick_slab",
     "minecraft:mossy_stone_brick_slab",
+    "minecraft:stone_slab",
+    "minecraft:stone_stairs",
```

Comment:
I think we should only have the stone brick variants, not the plain stone

Reply:
Yeah, the others are fine, let's remove those here then :shipit: